app/config/errors.php

```diff
@@ -519,6 +546,46 @@
         'description' => 'The project key has expired. Please generate a new key using the Appwrite console.',
         'code' => 401,
     ],
+    Exception::ROUTER_INVALID_TYPE => [
+        'name' => Exception::ROUTER_INVALID_TYPE,
+        'description' => 'Invalid domain configuration. Route type is not supported.',
+        'code' => 400,
+    ],
+    Exception::ROUTER_UNKNOWN_HOST => [
+        'name' => Exception::ROUTER_UNKNOWN_HOST,
+        'description' => 'Host is not trusted. Add a custom domain to your project first.',
+        'code' => 400,
+    ],
+    Exception::RULE_RESOURCE_ID_MISSING => [
+        'name' => Exception::RULE_RESOURCE_ID_MISSING,
+        'description' => 'With resourceType you provided, the resourceId is required.',
```

Comment:
Is it even possible for a resource ID to be missing? If the resource doesn't exist, we might want to throw a relevant 404 error like: `FUNCTION_NOT_FOUND`

Reply:
Great catch. Removing and using func nto found instead.